### PR TITLE
Add title for maintenance page, check response from maintenance page in an interval and determine if a page reload is necessary

### LINKED ISSUE #155: Let the maintenance page of the MaintenanceModeTask detect when the application / website is available again
And reload the application / website.

### PR DESCRIPTION
**Pull request by deborahvandervegt**
Closes #155

### CODE REVIEW

**Comment by niels-nijens** [comment]
Nice.

File: src/Resources/maintenance/index.html [hunk 1]
```diff
@@ -1,9 +1,24 @@
...
 <html>
     <head>
-        <title></title>
+        <title>In maintenance</title>
         <meta name='viewport' content='width=device-width, initial-scale=1'/>
         <meta name='theme-color' content='#4884be'>
+        <script type='text/javascript'>
+            var check = setInterval(checkForMaintentance, 10000);
+
+            function checkForMaintentance() {
```
Comment (niels-nijens) [inline src/Resources/maintenance/index.html]: Small typo checkForMaintentance -> checkForMaintenance

File: src/Resources/maintenance/index.html [hunk 2]
```diff
@@ -1,9 +1,24 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <title></title>
+        <title>In maintenance</title>
```
Comment (niels-nijens) [inline src/Resources/maintenance/index.html]: Could we change this to a more 'catchy' title? Any ideas?

Comment (deborahvandervegt) [inline src/Resources/maintenance/index.html]: "Catchy"? Like "Awesome stuff in progress..."? :-P

Comment (niels-nijens) [inline src/Resources/maintenance/index.html]: :smile: Maybe just the same as within the body: `We're currently upgrading our systems...`

**Comment by niels-nijens** [comment]
Thanks!

**Pull request merged**